content/blog/2024-07-25-guac-v0.8.0.md

```diff
@@ -0,0 +1,35 @@
+---
+title: GUAC v0.8.0 released
+layout: post
+authors: ["GUAC Maintainers"]
+date: 2024-07-25
+include_footer: true
+---
+
+[GUAC v0.8.0](https://github.com/guacsec/guac/releases/tag/v0.8.0) is now available.
+This release brings support for license information, node deletion, and many other improvements.
+You can now run vulnerability scans immediately on SBOM ingestion with the `--add-vuln-on-ingest` flag instead of waiting for the OSV certifier to run.
+To better represent the real world, the `isDependency` relationship now only exists on package versions instead of the package name.
+For a full list of changes, see the [release page on GitHub](https://github.com/guacsec/guac/releases/tag/v0.8.0).
+
+### License information support
+
+GUAC v0.8.0 adds support for parsing license information provided in [CycloneDX SBOMs](https://cyclonedx.org/use-cases/#license-compliance).
+The new release also includes a new experimental ClearlyDefined certifier.
+GUAC will query the [ClearlyDefined license data store](https://clearlydefined.io/) to discover license information for packages, even when the SBOM does not include that information.
+
+Although licenses don't directly impact security, they are an important part of understanding your software supply chain.
+We're excited to expand GUAC's capabilities in this area.
+
+### Node deletion
+
+GUAC v0.8.0 adds support for deleting the following evidence nodes:  `certifyVuln`, `hasSBOM`, and `hasSLSA`.
+This is helpful when SBOMs were ingested by accident or as part of a short-term demo.
+`Delete` is supported in both the graphQL client and the ENT backend.
```

Comment:
This will be key value and ent backend. Not graphQL client.

Reply:
Updated!